CMakeLists.txt

```diff
@@ -111,7 +114,7 @@ endif()
 # The reason for building ccl as a shared library is that we can link it to
 # class directly, and it's not a dependency anymore
 add_library(ccl SHARED $<TARGET_OBJECTS:objlib>)
-target_link_libraries(ccl ${GSL_LIBRARIES} ${FFTW_LIBRARIES} ${CLASS_LIBRARIES} ${ANGPOW_LIBRARIES} m)
+target_link_libraries(ccl ${GSL_LIBRARIES} ${FFTW_LIBRARIES} ${CLASS_LIBRARIES} ${ANGPOW_LIBRARIES} m $<$<BOOL:${OpenMP_C_FOUND}>:OpenMP::OpenMP_C>)
```

Comment:
OK, this notation is very obscure to me. What's going on?

Reply:
This is a generator expression (https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html). I wish there were an easier way to do this but with cmake, nothing ever seems easy to do. I added a comment on what it does.

This was in earlier versions already (e.g. 2.5.1) and just got restored here.